scripts/ibmcloud_post_install.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -e
+
+### Status flag
+
+log () {
+  FLAGFILE=/root/ibmcloud-post-install.log
+  echo "$(date) +++ $1" >> $FLAGFILE
+}
+
+log "Starting ibmcloud-post-install.sh"
+
+### SSH hardening
+### IBM Cloud machines by default allow SSH as root using the password. Given that the default one
+### is extremely simple, we disable this option.
+
+sed -i "s/.*RSAAuthentication.*/RSAAuthentication yes/g" /etc/ssh/sshd_config
+sed -i "s/.*PubkeyAuthentication.*/PubkeyAuthentication yes/g" /etc/ssh/sshd_config
+sed -i "s/.*PasswordAuthentication.*/PasswordAuthentication no/g" /etc/ssh/sshd_config
```

Comment:
Could pass the multiple replacements to a single sed command.

Reply:
Purely for readability, using multiple expressions in one will create a big one-liner difficult to modify in the future if someone wants to add one more parameter to change. With this here, you can just copy-paste and change a single one so `git blame` and stuff like this becomes easier